include/menoh/menoh.h

```diff
@@ -107,8 +107,7 @@ void MENOH_API menoh_delete_model_data(menoh_model_data_handle model_data);
  * This struct configure profiles of variables.
  *
  * See
- *  - menoh_variable_profile_table_builder_add_input_profile_dims_2()
- *  - menoh_variable_profile_table_builder_add_input_profile_dims_4()
+ *  - menoh_variable_profile_table_builder_add_input_profile()
```

Comment:
This commit removes old APIs but how about keep them as deprecated API for the time being to keep source/binary backward compatibility.

Reply:
I see. I'll return old interface.

Reply:
done.

Reply:
Thanks! :+1: